packages/buffetjs-custom/src/components/DateTime/index.js

```diff
@@ -78,9 +78,12 @@ function DateTime({
 
   useEffect(() => {
     if (!!value && moment(value).isValid()) {
-      const newDate = value._isAMomentObject === true ? value : moment(value);
-
-      setDateRef.current(newDate);
+      if (value._isAMomentObject) {
```

Comment:
This does not really fix the issue. Maybe we could try a different approach:

```
useEffect(() => {
  if (!isInit) {
    if (!!value && moment(value).isValid()) {
      const newDate = value._isAMomentObject === true ? value : moment(value);

      setTimeStamp(newDate)
   } 

  setInit(true);
 }
 

}, [isInit, value]);


```

Reply:
Thanks, I think both ways work. Yours is better, then we can remove the `setDateRef` here. I will make an update.

Reply:
I have made an update, the test case failed and I made a fix. please check if it works well